File: Cpp_Programming/Practice/LB_Class/program4.cpp
#include<iostream>
using namespace std;

int main()
{
    float fValue1 = 0.0f;                   //To store the first input
    float fValue2 = 0.0f;                   //To store the second input
    float fResult = 0.0f;                   //To store the result

    cout<<"Enter the first number : \n";
    cin>>fValue1;

    cout<<"Enter the second number : \n";
    cin>>fValue2;

    fResult = fValue1 + fValue2;

    cout<<"Addition is : %d\n", fResult;

    return 0;
}

File: Java_Programming/Practice/LB_class/program4.java
/*
    Algorithm
    START
        Accept first number as No1
        Accept second number as No2
        Perform addition of No1 & No2
        Display the result

    STOP
*/


package Practice.LB_class;

import java.util.Scanner;

class program4 
{
    public static void main(String A[])
    {
        //variable creation with default value
        float i = 0.0f, j = 0.0f, k = 0.0f;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the first number : ");
        i = sobj.nextInt();

        System.out.println("Enter the second number : ");
        j = sobj.nextInt();

        k = i + j;

        System.out.println("Addition is : "+k);

    }
}
